app/controllers/servicedown_controller.rb

```diff
@@ -0,0 +1,31 @@
+class ServicedownController < ActionController::Base
```

Comment:
(Rubocop) All controllers should subclass `ApplicationController` so that common configuration can be kept in one place. However, you specifically want to avoid the common configuration in this case.

```suggestion
# rubocop:disable Rails/ApplicationController
class ServicedownController < ActionController::Base
```

(see also the suggestion for the end of the file)

An alternative to this would be to split the existing configuration in `ApplicationController` into two levels but that seems to be overkill.